entity_accessors_test.go

```diff
@@ -49,7 +49,7 @@ func TestKeyValueEncoding(t *testing.T) {
 	// Write
 	httpWriter := httptest.NewRecorder()
 	//								Accept									Produces
-	resp := Response{httpWriter, "application/kv,*/*;q=0.8", []string{"application/kv"}, 0, 0, true, nil}
+	resp := Response{ResponseWriter: httpWriter, requestAccept: "application/kv,*/*;q=0.8", routeProduces: []string{"application/kv"}, prettyPrint: true}
```

Comment:
thank you for making these changes;  at the time of initial writing, I was lazy and then the arguments started growing...